charts/stable/firefly-iii/values.yaml

```
@@ -0,0 +1,462 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: fireflyiii/core
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+  # -- image tag
+  tag: version-5.6.14
+
+strategy:
+  type: Recreate
+
+cronjobs:
+  # Enable cronjob for creating recurring transactions
+  recurring:
+    enabled: false
+    schedule: "0 3 * * *"
```

```suggestion
    schedule: "0 3 * * *" # Schedule time in UTC
```
Worth mentioning to end users the cron will be scheduled in UTC time?